Cython/Compiler/Optimize.py

```diff
@@ -319,7 +319,27 @@ def _transform_reversed_iteration(self, node, reversed_function):
             return node
         elif len(args) > 1:
             error(reversed_function.pos,
-                  "reversed() takes exactly 1 argument")
+                  "reversed() takes exactly 1 argument")* Some more glossary entries
+* Some more glossary entries
+
+* Line length
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
+
+* Update docs/src/userguide/glossary.rst
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
+* Line length
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
+
+* Update docs/src/userguide/glossary.rst
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
+
+Co-authored-by: scoder <stefan_ml@behnel.de>
```

Comment:
This looks like a copy-paste error rather than code that's going to optimize stuff?

Reply:
… and you wouldn't believe how it got there …
Thanks for catching this.